packages/capabilities/package.json

```
@@ -0,0 +1,102 @@
+{
+  "name": "@web3-storage/capabilities",
+  "version": "0.0.0",
+  "description": "Capabilities for interacting with web3.storage",
```

Not sure this is better, but it makes bit more sense to me

```suggestion
  "description": "Capabilities provided by web3.storage",
```